class.jetpack-cli.php

```diff
@@ -1013,7 +1013,7 @@ public function partner_provision( $args, $named_args ) {
 		$body_json     = json_decode( wp_remote_retrieve_body( $result ) );
 
 		if( 200 !== $response_code ) {
-			if ( isset( $body_json->error ) ) {
+			if ( isset( $body_json->error ) && !is_wp_error( $body_json ) ) {
```

Comment:
This one wouldn't be as relevant as if `wp_remote_retrieve_body()` returns a WP_Error, then it shouldn't be one still after being passed through `json_decode()` in line 1013.